### ISSUE TITLE
Companies using MST

### ISSUE THREAD

**Issue by aMahdaoui** [mobxjs/mobx-state-tree#1078]
**_Question_**

* [ ] I've checked documentation and searched for existing issues
* [ ] I tried the [spectrum channel](https://spectrum.chat/?t=dad48299-3dfc-4e10-b6da-9af1e39498a3)

 
I made 2 prototypes to manage the state of one of our projects, the first using redux and the second using MST.
now we have to make a decision, which one to use, personally i prefer MST (easy to manage deep nested state, ...) but our CEO is asking me to look for companies that are already using MST, which i have not found so far
if anyone had an idea about that,
thanks for answering

**Comment by mweststrate** [comment]
DAZN and Infinite Red use it intensively, codesandbox.io, https://builderx.io/ are built using MST, and a quite impressive design tool of which I can't recall the at the moment :'). That is the ones I know from the top of my head in 30 secs :)

**Comment by aMahdaoui** [comment]
Yeah, i will add our company to the list when we start the project

**Comment by robskidmore** [comment]
We are using MST heavily at https://grow.com

**Comment by m3kka** [comment]
IBM iX London
Sorry can't really disclose the projects / clients but all quite high profile :)

We ❤️  `mobx-state-tree`

**Comment by dance-cmdr** [comment]
We are using MST in DAZN.

**Comment by antondomratchev** [comment]
We are using MST in an electron app and embedded widgets at Old Well Solutions. I've used it in several MVP products while at Grok Interactive. Its very fast to get started and is a great for rapid prototyping that you can scale. Much ❤️ for making this excellent tool

**Comment by llamaluvr** [comment]
Nudge checking in! Our React Native apps use MobX/ MST extensively, and I’m sure it will be all over our entire front end codebase before long. https://nudgecoach.com/

**Comment by southerneer** [comment]
[tinyrobot](https://tinyrobot.com/) uses (and loves) all flavors of mobx.

**Comment by LaurensBosscher** [comment]
[Simplicate](https://www.simplicate.nl/) will be using MST in the next version :)

**Comment by andraz** [comment]
MST is pretty low level, try searching for companies which are using some package which uses MST:
https://github.com/mobxjs/mobx-state-tree/network/dependents?dependent_type=PACKAGE

**Comment by mario-jerkovic** [comment]
Using MST in Rimac Automobili

**Comment by juFelipe** [comment]
Do someone knows if in SF there's a startup using MST looking for engineers?

**Comment by Relaxe111** [comment]
We are using MST heavily.

**Comment by chaudharykiran** [comment]
https://zegal.com/en-hk/ is using MST heavily in their all projects(old and new)

**Comment by tibotiber** [comment]
The whole studio for 3D modeling of spaces at [Smplrspace](https://www.smplrspace.com) is powered by MST, super happy with :)

**Comment by zuhair-naqvi** [comment]
Australian Open (React Native)

**Comment by jonbnewman** [comment]
https://payondelivery.com was made using MST

**Comment by MitchWeiss** [comment]
We've started using MST at www.airtasker.com and we are very happy with it! We migrated from Redux and Redux Observable and find it much simpler and quicker to add features with less boilerplate.

**Comment by sanketsahusoft** [comment]
We at [GeekyAnts](https://geekyants.com) have been using MST for a really long time. We totally love it. ❤️

**Comment by joshuajung** [comment]
My project www.meine-bewerbungen.de (German web app for tracking your job applications) is 100% MST in combination with Next.js – was and is a lot of fun to use with SSR. Hope to be able to pay back some of your effort some time.

**Comment by akatechis** [comment]
At [clarifai](https://www.clarifai.com/) we're completely rebuilding our react-redux SPA front-end using nextJS and MST.

**Comment by FakhruddinAbdi** [comment]
We also using MST at https://harzaan.com

**Comment by coolsoftwaretyler** [comment]
We use MST extensively at [WalterPicks](https://www.walterpicks.com/) for our React Native app.

I happen to know that ESPN also uses it, at least for their fantasy football drafting tool. I think they're build on Next.js (or perhaps some combination of Next and other React tooling).

**Comment by coolsoftwaretyler** [comment]
I'm going to convert this to a discussion. I think it's a little more appropriate over there!
